app/helpers/codelist_helper.rb

```diff
@@ -14,11 +14,26 @@ def yaml_to_objects(entity, type, with_empty_item = true)
   end
 
   def currency_select_options
-    objects = yaml_to_objects("generic", "default_currency", false)
+    objects = yaml_to_objects(entity: "generic", type: "default_currency", with_empty_item: false)
     objects.unshift(OpenStruct.new(name: "Pound Sterling", code: "GBP")).uniq
   end
 
-  def load_yaml(entity, type)
+  def region_select_options
```

Comment:
We were set to extend these activity options using a model method called `set_hierachy_defaults`. With this change I imagine we could later extend this to ingest a hierarchy object to use in a switch statement to deviate from the defaults where needed.